index.d.ts

```diff
@@ -163,12 +162,6 @@ declare module "skygear" {
     lambda(action: string, params: any): Promise<any>;
   }
 
-  export class DatabaseContainer {
-    constructor(container: Container);
-
-    uploadAsset(asset: Asset): Promise<Asset>;
-  }
-
```

Comment:
I prefer we keep DatabaseContainer because it is still available. https://github.com/SkygearIO/skygear-SDK-JS/blob/v1.7.2/packages/skygear-core/lib/database.js#L492

Reply:
updated, thanks.